pkg/apis/admissionregistration/types.go

```diff
@@ -316,8 +316,9 @@ type WebhookClientConfig struct {
 	// +optional
 	Service *ServiceReference
 
-	// `caBundle` is a PEM encoded CA bundle which will be used to validate the webhook's server certificate.
+	// `caBundle` is a PEM(Base64) encoded CA bundle which will be used to validate the webhook's server certificate.
```

Comment:
the doc on the `admissionregistration/v1beta1/types.go` file looks better... let's go with that